printer_mods/ZZP8202/Pull-out_electrical_store/README.md

```diff
@@ -0,0 +1,42 @@
+# Voron 2.4 Pull-out electrical Store 抽拉式电器仓
+
+## 适用
+
+| Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW | Trident |

```

Comment:
this doesn't need to go in your readme file. it's only for the main one in the printer_mods directory.